jenkins-pipelines/longevity-counters-multidc.jenkinsfile

```diff
@@ -5,7 +5,7 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 longevityPipeline(
     backend: 'aws',
-    availability_zone: 'a',
+    availability_zone: 'a,b,c',
```

Comment:
so, we rollback this test to be true multi-az?

Reply:
no, it's for testing, I wanted to see it working in real AZs (not the simulated)